index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <nav class="sticky">
+    <div class="main-choose"><a href="#welcome-section">About</a></div>
+    <div class="main-choose"><a href="#projects">Work</a></div>
+    <div class="main-choose"><a href="#footer">Contact</a></div>
+  </nav>
+
+  <div id="welcome-section">
+    <h1 class="h1-welcome-section">Hey I am Mimic</h1>
+    <p class="p-welcome-section"> a web developer</p>
+  </div>
+
+  <div id="projects">
+    <h1 class="h1-projects">These are some of my projects</h1>
+    <hr size="5px" color="#fff" width="50%" style="margin-left: 25%;">
+    <div id="projects-div-one">
+      <div class="project">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg">
+        <p class="caption"><a href="https://mahmoudhesham009.github.io/MicroversFirstProject/">Tribute Page</a></p>
+      </div>
+      <div class="project">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/random-quote-machine.png">
+        <p class="caption"><a href="https://mahmoudhesham009.github.io/MicroversFirstProject/">Tribute Page</a></p>
+      </div>
+      <div class="project">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png">
+        <p class="caption"><a href="https://mahmoudhesham009.github.io/MicroversFirstProject/">Tribute Page</a></p>
+      </div>
+    </div>
+    <div id="projects-div-two">
+      <div class="project">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png">
+        <p class="caption"><a href="https://mahmoudhesham009.github.io/MicroversFirstProject/">Tribute Page</a></p>
+      </div>
+      <div class="project">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/wiki.png">
+        <p class="caption"><a href="https://mahmoudhesham009.github.io/MicroversFirstProject/">Tribute Page</a></p>
+      </div>
+      <div class="project">
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tic-tac-toe.png">
+        <p class="caption"><a href="https://mahmoudhesham009.github.io/MicroversFirstProject/">Tribute Page</a></p>
+      </div>
+    </div>
+  </div>
+
+
+  <div id="navbar">
+    <h1 class="h1-navbar">
+      Let's work together...
+    </h1>
+    <p class="p-navbar">How do you take your coffee?</p>
+
+    <div id="links">
+      <a class="social-media" href="https://www.facebook.com">FaceBook</a>
+      <a class="social-media" href="https://www.github.com">GitHub</a>
+      <a class="social-media" href="https://www.twitter.com">Twitter</a>
+      <a class="social-media" href="email@gmail.com">Send an email</a>
+      <a class="social-media" href="+2123456789">Call me</a>
+    </div>
```

Comment:
- [ ] As I mentioned above, Please use list `<ul>` or `<ol>` for the social links, I think you can put these links as a list of items. Whenever we have a list of similar or identical elements, we should group them together on a list. It is also better for SEOs interpretation. 😉